src/App.js

```diff
@@ -1,21 +1,41 @@
+import {useState} from 'react';
 import styled from 'styled-components';
 
-import ChairCard from './component/ChairCard';
+import ChairCard from './component/ChairCard/ChairCard';
 import data from './data.json';
 
-
-
 export default function App() {
+const chairsData = data;
+  const [showDescription, setShowDescription] = useState(false);
+
+  /* function toggleDescription(id) {
+  setChairsData(chairsData
+      .map(item1 => ({...item1, showDescription: false}))
+      .map(item2 => (item2._id === id ? {...item2, showDescription: !item2.showDescription} : item2)));
+    
+  }
+  */
 
+  const toggleDescription = index => {
+    showDescription === index ? setShowDescription(false) : setShowDescription(index);
```

Comment:
It is better to work with the id at this point instead of the index. The index order could be changed, the id is unequivocally.